ci-operator/step-registry/telco5g/origin/tests/telco5g-origin-tests-commands.sh

```diff
@@ -44,30 +41,236 @@ git clone -b "${CNF_BRANCH}" "${CNF_REPO}" cnf-features-deploy
+if [ -n "${TEST_SKIPS}" ]; then
+    grep -v "$TEST_SKIPS" "$ARTIFACT_DIR/tests-all.txt" > "$ARTIFACT_DIR/tests-run.txt"
+
+    if ! grep "$TEST_SKIPS" "$ARTIFACT_DIR/tests-all.txt" > "$ARTIFACT_DIR/tests-skip.txt"; then
+      echo >&2 "ERROR: No tests were found matching the TEST_SKIPS regex:"
+      echo >&2 "$TEST_SKIPS"
+      exit 1
+    fi
+
+    TEST_FILE=tests-run.txt
+else
+    TEST_FILE=tests-all.txt
+fi
+
+IFS=- read -r CLUSTER_NAME _ <<< "$(cat "${SHARED_DIR}/cluster_name")"
+
+cat << EOF > run-openshift-tests.yml
+---
+- name: Run OpenShift's conformance test suite
+  hosts: hypervisor
+  gather_facts: false
+  any_errors_fatal: true
+
+  tasks:
+  - name: Copy OpenShift's conformance test suite through bastion host to installer, run it and retrieve results
+    block:
+    - name: Create script to run openshift-tests binary on bastion host
+      ansible.builtin.copy:
+        content: |
+          #!/bin/bash
+
+          set -o nounset
+          set -o errexit
+          set -o pipefail
+
+          cd /home/cloud-user/
+
+          export KUBECONFIG=kubeconfig
+
+          ./openshift-tests run "${TEST_SUITE}" \\
+            --provider '${TEST_PROVIDER}' \\
+            -o e2e.log --junit-dir junit --file tests.txt \\
+            > openshift-tests.log
+        dest: '/tmp/kcli_${CLUSTER_NAME}_run-openshift-tests.sh'
+        mode: 0755
+
+    - name: Copy openshift-tests binary to bastion host
+      ansible.builtin.copy:
+        src: '${PWD}/openshift-tests'
+        dest: '/tmp/kcli_${CLUSTER_NAME}_openshift-tests'
+
+    - name: Copy list of tests to bastion host
+      ansible.builtin.copy:
+        src: '${ARTIFACT_DIR}/${TEST_FILE}'
+        dest: '/tmp/kcli_${CLUSTER_NAME}_${TEST_FILE}'
+
+    - name: Copy script to run openshift-tests binary to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_run-openshift-tests.sh' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/run-openshift-tests.sh' \\
+          && kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' 'chmod a+x /home/cloud-user/run-openshift-tests.sh'
+
+    - name: Copy openshift-tests binary to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_openshift-tests' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/openshift-tests' \\
+          && kcli ssh 'cloud-user@${CLUSTER_NAME}-installer' 'chmod a+x /home/cloud-user/openshift-tests'
+
+    - name: Copy list of tests to installer
+      ansible.builtin.shell:
+        cmd: |
+          kcli scp '/tmp/kcli_${CLUSTER_NAME}_${TEST_FILE}' \\
+            'cloud-user@${CLUSTER_NAME}-installer:/home/cloud-user/tests.txt' \\
+
+    - name: Fix permissions for kubectl on installer
+      ansible.builtin.shell:
+        cmd: kcli ssh 'root@${CLUSTER_NAME}-installer' 'chmod a+x /usr/bin/kubectl'
```

Comment:
I think it's executable for sure, is there a problem with it?
Why no to use `oc` btw?

Reply:
`openshift-tests` uses `kubectl` but `kubectl` in installer is not marked executable. This has to be fixed in the code which sets up the installer but to move fast i added this workaround here.